server/webapp/WEB-INF/rails.new/app/presenters/api_v3/plugin/authorization_plugin_info_representer.rb

```diff
@@ -15,9 +15,19 @@
 ##########################################################################
 module ApiV3
   module Plugin
-    class AuthorizationPluginInfoRepresenter < BasePluginInfoRepresenter
+    class AuthorizationPluginInfoRepresenter < BaseRepresenter
+      alias_method :plugin, :represented
+
       link :image do |opts|
-        opts[:url_builder].plugin_images_url(plugin_id: id, hash: plugin.getImage.getHash()) if plugin.image
+        opts[:url_builder].plugin_images_url(plugin_id: plugin.getDescriptor.id, hash: plugin.getImage.getHash()) if plugin.image
+      end
+
+      link :auth_config_doc do |opts|
```

Comment:
This will end up adding `_links` at multiple places, I think for a resource we should have all the links under one attribute